segmenters/nlp/Sentencizer/__init__.py

```diff
@@ -14,8 +14,9 @@ class Sentencizer(BaseSegmenter):
     :param min_sent_len: the minimal number of characters,
         (including white spaces) of the sentence, by default 1.
     :param max_sent_len: the maximal number of characters,
-        (including white spaces) of the sentence, by default 1e5.
-    :param punct_chars: the punctuation characters to split on.
+        (including white spaces) of the sentence, by default 512.
+    :param punct_chars: the punctuation characters to split on,
```

Comment:
Actually, the problem is that the `type` hint is wrong, it should be `Optional[Tuple[str]]`

Reply:
What you mean? for `max_sent_len`? this is an `int` right?

Reply:
the punct_chars I mean

Reply:
Ah, but isn't the type `List[str]`?

Reply:
well it says it is `str`

Reply:
Yeah that's wrong, but I was wondering for the type `Optional[Tuple[str]]` you mentioned

Reply:
Optional[List[str]] is good also. Optional indicates that a value of None is properly handled qnd since None is the default parameter let's add it here